specs/fault-proof/stage-one/optimism-portal.md

```diff
@@ -251,9 +321,36 @@ has not been initialized then this value will be `address(0)` and should not be
 `OptimismPortal` is not currently executing an withdrawal transaction then this value will be
 `0x000000000000000000000000000000000000dEaD` and should not be used.
 
-### proveWithdrawalTransaction
+### proveWithdrawalTransaction (Super Roots)
+
+Allows a user to [prove](#proven-withdrawal) a withdrawal transaction within an `OptimismPortal`
+that uses dispute games that argue over [Super Roots](#super-root).
+
+- MUST revert if the withdrawal target is the address of the `OptimismPortal` itself.
+- MUST revert if the withdrawal is being proven against a game that is not a
+  [Proper Game](./anchor-state-registry.md#proper-game).
+- MUST revert if the withdrawal is being proven against a game that is not a
+  [Respected Game](./anchor-state-registry.md#respected-game).
+- MUST revert if the withdrawal is being proven against a game that has resolved in favor of the
+  Challenger.
+- MUST revert if `superRootsActive` is `false`.
+- MUST revert if the proof provided by the user of the contents of the Super Root that the dispute
+  game argues about is invalid. This proof is verified by hashing the user-provided Super Root
+  contents and comparing them to the Super Root in the referenced dispute game. The user provides
+  a specific Output Root index to extract and the chain ID of this Output Root must match the chain
+  ID within the `SystemConfig` contract.
```

Comment:
Can we split up the first / second part of this invariant? They are checked separately, would be a bit easier to read.